scripts/vet.sh

```diff
@@ -184,4 +184,7 @@ revive -formatter plain ./... >"${REV_OUT}" || true
 # TODO: Remove `|| true` to unskip linter failures once existing issues are fixed.
 (noret_grep -v "unused-parameter" "${REV_OUT}" | not grep -v "\.pb\.go:") || true
 
+# Error if trailing spaces found in any files excluding files in .git directory and *.md files
+$(find . -path ./.git -prune -o -type f ! -name "*.md" -exec grep -Hn "[[:blank:]]$" {} \;) | fail_on_output
```

Comment:
Why exclude markdown files? I think trailing whitespaces should always be disallowed?

Reply:
FIxed